### PR TITLE
Update SES Version - Vulnerability Fix

### PR DESCRIPTION
**Pull request by andrewpeters9**
## Explanation

* Vulnerability has been found recently with `SES<0.16.0` - https://github.com/advisories/GHSA-whpx-q3rq-w8jc
*  Vulnerable versions of the packages are currently loaded in via 
  a) directly via the project's workspace and 
  b) via `@metamask/phishing-warning`
* Package changelog: https://github.com/endojs/endo/blob/master/packages/ses/CHANGELOG.md

## Manual Testing Steps
* `yarn && yarn start`
* Open a URL within the `@metamask/phishing-warning` blocked-list to ensure that the plugin is working as intended
  * Example: https://thekrakenz.com/
* Open the extension in tab form, inspect to see that `lockdown-install.js` is still loaded - i.e., the SES code is still loaded
  * https://gyazo.com/9de8e66a6de39f1a6d9bef02f6b336c7

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue

### CODE REVIEW

File: app/scripts/lockdown-run.js
Comment (andrewpeters9) [inline app/scripts/lockdown-run.js:9]: Added due to this breaking change:

https://github.com/endojs/endo/blob/master/packages/ses/CHANGELOG.md#0150-2021-11-02

**Comment by andrewpeters9** [comment]
I have read the CLA Document and I hereby sign the CLA

File: test/helpers/protect-intrinsics-helpers.js
Comment (andrewpeters9) [inline test/helpers/protect-intrinsics-helpers.js:20]: `@agoric/*` packages no longer pulled in due to the version change

File: package.json
Comment (Gudahtt) [inline package.json:94]: v1.2.2 of the phishing warning page has just been published - you can update that here to point the extension at the new page, and drop this resolution

**Review by Gudahtt — DISMISSED** [review]
LGTM!

Though we'll want to flag this to QA as needing thorough regression testing

**Comment by andrewpeters9** [comment]
> LGTM!
> 
> Though we'll want to flag this to QA as needing thorough regression testing

Thanks. I've added more testing info and I've added the QA label. Is there anything else I need to do? @Gudahtt

**Comment by Gudahtt** [comment]
Thanks! No further action needed as far as I'm aware.

**Review by jpuri — DISMISSED** [review]

**Review by pedronfigueiredo — DISMISSED** [review]

**Review by Gudahtt — DISMISSED** [review]

**Comment by andrewpeters9** [comment]
@Gudahtt Am I free to merge this directly after two approvals? Or does the QA team need to test it on the branch before it gets merged?

**Review by legobeat — APPROVED** [review]

**Review by Gudahtt — APPROVED** [review]

**Pull request merged**
